src/containers/sidebar/sections/tools/index.tsx

```diff
@@ -1,22 +1,23 @@
-import classNames from 'classnames';
+import cn from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import { ButtonRuler } from '../../../map/tool-ruler';
 
-import './style.css';
+import styles from './tools.module.css';
 
 type Props = {
   className?: string;
```

Comment:
что бы не было путаницы, что за класс
```suggestion
  extraClass?: string;
```